.travis.yml

```diff
@@ -93,6 +93,7 @@ jobs:
         - make install
         # - make package
         # - dpkg -i openPMD*.deb
+        - ls -R $HOME/openPMD-test-install | grep ":$" | sed -e 's/:$//' -e 's/[^-][^\/]*\//--/g' -e 's/^/   /' -e 's/-/|/'
```

Comment:
We can probably keep this for travis long-term. Having 10 extra trailing lines won't hurt.